packages/react/src/dialog/Dialog.tsx

```diff
@@ -0,0 +1,60 @@
+import * as RadixDialog from "@radix-ui/react-dialog";
+import { type ComponentPropsWithRef, type ReactNode, forwardRef } from "react";
+
+import type { ExtendProps } from "../utils";
+
+import { Box } from "../box";
+import { Button } from "../button";
+import { Flex } from "../flex";
+import { Paper } from "../paper";
+import { CloseIcon } from "./CloseIcon";
+import * as styles from "./Dialog.css";
+
+type DialogProps = ExtendProps<
+  ComponentPropsWithRef<typeof Box>,
+  ComponentPropsWithRef<typeof RadixDialog.Root>,
```

Comment:
let's pick out the specific props we want from the root
```suggestion
  Pick<ComponentPropsWithRef<typeof RadixDialog.Root>, "open" | "modal">,
```